stellar-core/debian/control

```diff
@@ -8,14 +8,14 @@ Homepage: https://www.stellar.org
 
 Package: stellar-core
 Architecture: any
-Depends: libc++1-8, libc++abi1-8, libpq5, libsqlite3-0 (>=3.11.0), stellar-core-utils
-Recommends: stellar-core-utils, stellar-core-prometheus-exporter
-Description: Stellar  is  a  decentralized, federated peer-to-peer network
- Stellar  is  a  decentralized, federated peer-to-peer network that allows people to send payments in any asset anywhere in the world instantaneously, and with minimal fee.
+Depends: libsqlite3-0 (>=3.11.0), stellar-core-utils, ${shlibs:Depends}, ${misc:Depends}
```

Comment:
Will `${shlibs:Depends}` and `${misc:Depends}` derive the right clang version from `Build-Depends` above?

Reply:
It will derive versions from the binary. The idea is that dependencies will be calculated at packaging time and will match OS version where compilation and packaging happens.

Extra info if you're interested:
https://manpages.debian.org/testing/dpkg-dev/dpkg-shlibdeps.1.en.html